scripts/git-hooks/pre-commit

```diff
@@ -1,3 +1,8 @@
 #!/bin/sh
 
 make lint-git
+
+STAGED_MOVE_FILES=$(git diff --cached --name-only --diff-filter=ACM | grep '\.move$')
+
```

Comment:
should we check `if stage_move_files not empty then:` To make sure we dont run linter unecessarly. There might be some side effects